kubeflow-katib.yaml

```diff
@@ -0,0 +1,80 @@
+package:
+  epoch: 0
+  name: kubeflow-katib
+  version: 0.15.0
+  description: Kubeflow Katib services
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - wget
+      - ca-certificates
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+      - build-base
+      - python-3.10-dev
+      - py3.10-pip
+      - gfortran
+      - pcre-dev
+      # libopenblas-dev liblapack-dev not sure if these are needed, builds fine without.  From https://github.com/kubeflow/katib/blob/b9dc63efb55bec19bdd80ca6f2930a65795f7c41/cmd/earlystopping/medianstop/v1beta1/Dockerfile#L10
+
+data:
+  - name: go-builds
+    items:
+      cert-generator: cert-generator/v1beta1
+      katib-controller: katib-controller/v1beta1
+      db-manager: db-manager/v1beta1
+      metricscollector: metricscollector/v1beta1/file-metricscollector
+      suggestion-goptuna: suggestion/goptuna/v1beta1
+
+  - name: python-builds
+    items:
+      earlystopping: earlystopping/medianstop/v1beta1
+      metricscollector: metricscollector/v1beta1/tfevent-metricscollector
+      suggestion-hyperband: suggestion/hyperband/v1beta1
+      suggestion-hyperopt: suggestion/hyperopt/v1beta1
+      suggestion-nas-darts: suggestion/nas/darts/v1beta1
+      suggestion-nas-enas: suggestion/nas/enas/v1beta1
+      suggestion-optuna-enas: suggestion/optuna/v1beta1
+      suggestion-pbt-enas: suggestion/pbt/v1beta1
+      suggestion-skopt-enas: suggestion/skopt/v1beta1
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/kubeflow/katib
+      expected-commit: 05ab6f012ab6723609700a47a6cfa1845080e9b6
+      tag: v${{package.version}}
+
+subpackages:
```

Comment:
I initially added pipelines to the subpackages, but now wondering if we should build all the packages in the main melange pipeline, `strip`  them and then `mv` the binaries in the subpackages from the `destdir` to `subpkgdir`

Reply:
range only works for for subpackages